smi-conformance/grpc/handlers.go

```diff
@@ -56,15 +57,20 @@ func (s *Service) RunTest(ctx context.Context, req *conformance.Request) (*confo
 	}
 
 	result := test_gen.RunTest(config, req.Mesh.Annotations, req.Mesh.Labels)
-	totalcases := 3
-	failures := 0
+	totalSteps := 24
```

Comment:
Is this true for every mesh?

Reply:
Aye. That are the total test Steps we are checking. Coincides with the no. of tests in [layer5.io](https://layer5.io/projects/service-mesh-interface-conformance).